backend/src/main/java/com/empire/NationData.java

```diff
@@ -82,7 +106,7 @@ static Ideology getStateReligion(String kingdom, World w) {
 	String colorBg;
 	Culture culture;
 	List<Integer> coreRegions = new ArrayList<>();
-	private ArrayList<String> tags = new ArrayList<>();
+	private ArrayList<Tag> tags = new ArrayList<>();
```

Comment:
Should type this to the List interface instead of the implementation.

Reply:
Done.